src/main/java/org/corfudb/infrastructure/log/RollingLog.java

```diff
@@ -39,10 +36,17 @@
 public class RollingLog extends AbstractLocalLog {
 
     private Map<Long, FileHandle> channelMap;
+    private String pathSeparator;
 
     public RollingLog(long start, long end, String path, boolean sync) {
         super(start, end, path, sync);
         channelMap = new HashMap<>();
+        if (File.pathSeparator.equals(":")) {
+            pathSeparator = "/";     // OS X ... use slash, please
```

Comment:
Code elsewhere in this module assumes that File.pathSeparator = UNIX/Linux path separator, slash.  On OS X, I discovered that it is colon.  I assumed that we want to use slash and rely on OS X's POSIXy path translation to manage the colon directory separators deep inside libc and/or system calls.